### PR TITLE
refactor(parbase): Improve logging in FairParSet

### PR DESCRIPTION
**Pull request by ChristianTackeGSI**
And re-arrange some if statements

See-Also: #1541

Included one commit from #1552

---

Checklist:

* [X] Followed the [Contributing Guidelines](https://github.com/FairRootGroup/FairRoot/blob/dev/CONTRIBUTING.md)

### CODE REVIEW

File: fairroot/parbase/FairParSet.cxx [hunk 1]
```diff
@@ -47,30 +47,34 @@ Bool_t FairParSet::init()
...
 
     Bool_t allFound = kFALSE;
-    FairParIo* io = 0;
-    if (rtdb) {
-        io = rtdb->getFirstInput();
+    FairParIo* io = rtdb->getFirstInput();
+    if (io) {
+        allFound = init(io);
+    } else {
+        LOG(warn) << "FairParSet::init()/" << GetName() << ": No first input";
```
Comment (karabowi) [inline fairroot/parbase/FairParSet.cxx:60]: This warning (just as the second one) is unnecessary and would be too verbose.
The users may (from what I recall) use `First` or `Second` input independently.

Comment (ChristianTackeGSI) [reply fairroot/parbase/FairParSet.cxx:60]: Okay, I'll turn it into a `debug`, okay?

Comment (ChristianTackeGSI) [reply fairroot/parbase/FairParSet.cxx]: I factored the "which input is there and not" into the FairRuntimeDb::initContainers caller, which now lists the available inputs.

**Comment by ChristianTackeGSI** [comment]
@karabowi I included one of your commits from #1552.

I have also added even more `LOG(info)` instead of `cout`.

A current, typical output can be seen for example here:
https://cdash.gsi.de/testDetails.php?test=20858392&build=463833

**Review by karabowi — APPROVED** [review]

**Pull request merged**